app/models/graph-controller.ts

```diff
@@ -81,10 +81,10 @@ export class GraphController {
         }
 
         // TODO: ...like
+        const likeVal = new ElementValue(this.activeEdgeType, this.core.environment);
 
-        return validateEdge(this.plugin, source, destination, undefined);
+        return validateEdge(this.plugin, source, destination, likeVal);
```

Comment:
I've added an addition check if a bridge exists and only use the active edge type if one doesn't exist in my latest commit.